MoreLinq.Test/OrderedMergeTest.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace MoreLinq.Test
+{
+    [TestFixture]
+    class OrderedMergeTest
```

Comment:
Missing `public` modifier. While not strictly needed, we might as well put it to be consistent with the rest of the test fixture classes.